inference-engine/src/low_precision_transformations/src/pull_transpose_through_dequantization.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "low_precision/pull_transpose_through_dequantization.hpp"
+
+#include <memory>
+#include <queue>
+#include <vector>
+
+#include <ngraph/opsets/opset1.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+#include <ngraph/pattern/op/or.hpp>
+#include "low_precision/network_helper.hpp"
+
+using namespace ngraph;
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::low_precision::PullTransposeThroughDequantization, "PullTransposeThroughDequantization", 0);
+
+namespace pull_transpose_through_dequantization {
+
+std::shared_ptr<Node> moveThroughElementwise(const std::shared_ptr<Node>& transpose, const std::shared_ptr<Node>& elementwise) {
+    const auto transposeValues = transpose->get_input_node_shared_ptr(1);
+    NGRAPH_CHECK(transposeValues != nullptr, "transpose constant was not found");
+
+    const auto constantIndex = ngraph::pass::low_precision::NetworkHelper::getConstantInputIndex(elementwise);
+    NGRAPH_CHECK(constantIndex != -1);
+
+    const auto elementwiseValues = elementwise->get_input_node_shared_ptr(constantIndex);
+
+    const std::shared_ptr<opset1::Transpose> newTranspose = as_type_ptr<opset1::Transpose>(transpose->clone_with_new_inputs({
```

Comment:
Are we sure that eltwise doesn't perform broadcasting underneath?

Reply:
fixed, done, thanks